util/cmake/FindSeqAn.cmake

```diff
@@ -174,6 +171,8 @@ endif ()
 # Require C++14
 # ----------------------------------------------------------------------------
 
+# The standard library of visual studio defines __cplusplus still as
```

Comment:
How would you change the comment? I find it without a comment not clear, why we exclude MSVC.